README.md

```diff
@@ -10,6 +10,15 @@ Taking care of:
 
 100% Swift.
 
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | Maven  |
```

Comment:
Noticed forgot to change `Maven` build ;-)

What would be the best way to describe the build system here?